### ISSUE TITLE
Tests Pass Even When Dependencies Are Missing

### ISSUE THREAD

**Issue by nikicc** [conda-forge/orange3-text-feedstock#5]
With the introduction of the `Import Documents` widget, we got four new dependencies. After bumping the version and updating the hash value, all tests passed although the new dependencies were missing and the widget wouldn't be able to run.

We should update the tests to catch such cases.

**Comment by astaric** [comment]
Orange feedstock uses
```
- python -c "import pkg_resources; pkg_resources.require('Orange3')"
```
to check that all requirements specified in setup.py are installed.

**Comment by nikicc** [comment]
This was already fixed in #6. Seems like we forgot to close the issue 🙈
